### PR TITLE
M54 rebasing. Adapt to CLs in the upstream

### PR DESCRIPTION
**Pull request by msisov**

### CODE REVIEW

**Comment by rakuco** [comment]
You don't seem to be using the latest `next` branch: see how there are several duplicate commits above.

**Comment by msisov** [comment]
please review now

File: runtime/browser/runtime_geolocation_permission_context.cc [hunk 1]
```diff
@@ -13,10 +13,10 @@
 #include "components/url_formatter/url_formatter.h"
 #include "components/user_prefs/user_prefs.h"
 #include "content/public/browser/browser_thread.h"
-#include "content/public/browser/geolocation_provider.h"
 #include "content/public/browser/render_process_host.h"
 #include "content/public/browser/render_view_host.h"
 #include "content/public/browser/web_contents.h"
+#include "device/geolocation/geolocation_provider.h""
```
Comment (rakuco) [inline runtime/browser/runtime_geolocation_permission_context.cc]: There's an extra " at the end here.

**Comment by msisov** [comment]
fixed

**Pull request merged**